docs/examples/Naive_Classification.py

```diff
@@ -0,0 +1,188 @@
+# it stays a bus).
+# This is modelled in the state space by 3 state vector elements. The first component indicates
+# the probability that the target is a bicycle, the second that it is a car, and the third that it
+# is a bus.
+# For this model, it will remain a bus, therefore the 3rd state vector component will be
+# :math:`1` throughout, with the other components remaining :math:`0`.
+
+import numpy as np
+from stonesoup.types.state import State, StateVector
+
+np.random.seed(1991)
+
+gt = []
+nsteps=100
+
+for step in range(nsteps):
+    gt.append(State(StateVector([0, 0, 1])))  # bicycle, car, bus
+
+
+# %%
+# Modelling state transition
+# --------------------------
+# We require a model of the state transition in order to track the target's classification.
+# This will be a simple linear map, represented by a :math:`3\times 3` matrix with no noise added.
+# We might not know that the target does not switch classification, so the matrix should not simply
+# be the identity matrix.
+#
+# .. math::
+#       T = \begin{bmatrix}
+#  		P(Bicycle_{t}|Bicycle_{t-1}) & P(Bicycle_{t}|Car_{t-1}) & P(Bicycle_{t}|Bus_{t-1}) \\
+#  		P(Car_{t}|Bicycle_{t-1}) & P(Car_{t}|Car_{t-1}) & P(Car_{t}|Bus_{t-1}) \\
+#  		P(Bus_{t}|Bicycle_{t-1}) & P(Bus_{t}|Car_{t-1}) & P(Bus_{t}|Bus_{t-1})
+#  	    \end{bmatrix}
+#
+# The transitioned state is given by
+# :math:`f(X_{t-1})_{j} = T_{ij}{X_{t-1}}_{i} = T^T_{ji}{X_{t-1}}_{i}`.
+# Therefore, we take the transpose :math:`f(X_{t-1}) = T^{T}X_{t-1}` when transitioning the state.
+
+F = np.array([[0.8, 0.1, 0.1],
+              [0.1, 0.8, 0.1],
+              [0, 0.1, 0.9]])
+
+
+def transit(state):
+    return F.T @ state.state_vector
+
+
+# %%
+# Modelling observations
+# ----------------------
+# Observations of the target may be incorrect, and affected by various input from a sensor.
+# We simply model an observer of the target's size, with an understanding of the underlying
+# distribution of bicycle, car and bus targets that are/could be small, medium or large.
+# This is defined in the emission matrix, utilised by the observer model to determine a measurement
+# which just says whether the target is small, medium or large. Similar to the state space
+# categories, this is represented by a detection state vector with 3 components, either 0 or 1.
+# To determine which classification is observed, we randomly sample from the multinomial
+# distribution defined by the row of the emission matrix corresponding to the most likely state of
+# the target.
+#
+# .. math::
+#       E = \begin{bmatrix}
+# 		P(Small | Bicycle) & P(Medium | Bicycle) & P(Large | Bicycle)\\
+# 		P(Small | Car) & P(Medium | Car) & P(Large | Car)\\
+# 		P(Small | Bus) & P(Medium | Bus) & P(Large | Bus)
+# 	    \end{bmatrix}
+#
+
+import scipy
+
+E = np.array([[0.89, 0.1, 0.01],
+              [0.3, 0.3, 0.4],
+              [0.1, 0.3, 0.6]])
+
+# ie. 89% of bicycles are small, 10% are medium, 1% are large.
```

Comment:
It would be more accurate to say that bicycles return measurements = small 89% of the time, medium 10% and large 1%. That might be because 1% of bicycles are large or (more likely) because sensor isn't perfect.